docs/basic-features/script.md

```diff
@@ -13,102 +24,135 @@ description: Next.js helps you optimize loading third-party scripts with the bui
+## Overview
 
-With `next/script`, you can define the `strategy` property and Next.js will optimize loading for the script:
+Websites often use third-party scripts to include different types of functionality into their site, such as analytics, ads, customer support widgets, and consent management. However, this can introduce problems that impact both user and developer experience:
 
-- `beforeInteractive`: For critical scripts that need to be fetched and executed **before** the page is interactive, such as bot detection and consent management. These scripts are injected into the initial HTML from the server and run before self-bundled JavaScript is executed.
-- `afterInteractive` (**default**): For scripts that can fetch and execute **after** the page is interactive, such as tag managers and analytics. These scripts are injected on the client-side and will run after hydration.
-- `lazyOnload` For scripts that can wait to load during idle time, such as chat support and social media widgets.
+- Some third-party scripts are heavy on loading performance and can drag down the user experience, especially if they are render-blocking and delay any page content from loading
+- Developers often struggle to decide where to place third-party scripts in an application to ensure optimal loading
 
-> **Note:**
->
-> - `<Script>` supports inline scripts with `afterInteractive` and `lazyOnload` strategy.
-> - Inline scripts wrapped with `<Script>` _require an `id` attribute to be defined_ to track and optimize the script.
+The Script component makes it easier for developers to place a third-party script anywhere in their application while taking care of optimizing its loading strategy.
 
 ## Usage
 
-Previously, you needed to define `script` tags inside the `Head` of your Next.js page.
+To add a third-party script to your application, import the `next/script` component:
 
-```js
-// Before
+```jsx
+import Script from 'next/script'
+```
 
-// pages/index.js
-import Head from 'next/head'
+### Strategy
 
-export default function Home() {
-  return (
-    <>
-      <Head>
-        <script async src="https://www.google-analytics.com/analytics.js" />
-      </Head>
-    </>
-  )
-}
+With `next/script`, you decide when to load your third-party script by using the `strategy` property:
+
+```jsx
+<Script src="https://connect.facebook.net/en_US/sdk.js" strategy="lazyOnload" />
 ```
 
-Now, you use `next/script` in the body of your Next.js page. It has client-side functionality that decides when and how to load the remote script based on the `strategy`.
+There are three different loading strategies that can be used:
 
-> **Note:**
->
-> - `next/script` **must not** be placed in either a `next/head` component or in `pages/_document.js`.
```

Comment:
Hi, was the Script component always able to be placed inside of `next/head` of `_document.js`? Or was this achieved via a change to the Script component in a recent release?